aem-authoring-toolkit-samples/toolkit-samples-core/src/main/java/com/exadel/aem/toolkit/samples/models/WarriorDescriptionComponent.java

```diff
@@ -182,12 +182,12 @@
     private boolean isLikesFilms;
 
     @Hidden
-    @DependsOn(query = "@parentPath", action = "getParentColorTheme")
+    @DependsOn(query = "'../../colorTheme'", action = "get-property")
     @DependsOnRef(name = "isDarkColorTheme", type = DependsOnRefTypes.BOOLSTRING)
-    private boolean isDarkColorTheme;
+    private String isDarkColorTheme;
 
     @PlaceOnTab(WarriorDescriptionComponent.TAB_FILMS)
-    @DependsOn(query = "@isDarkColorTheme", action = "namespaceFilter")
+    @DependsOn(query = "@isDarkColorTheme === 'true'", action = "namespaceFilter")
```

Comment:
Seems to be incorrect.
Does it work properly with the type = DependsOnRefTypes.BOOLSTRING ?